core/src/main/java/org/infinispan/conflict/resolution/EntryMergePolicy.java

```diff
@@ -0,0 +1,14 @@
+package org.infinispan.conflict.resolution;
+
+import org.infinispan.commons.util.Experimental;
+import org.infinispan.container.entries.CacheEntry;
+
+/**
+ * The policy to be applied to detected conflicts on partition merges.
+ *
+ * @author Ryan Emerson
+ */
+@Experimental
+public interface EntryMergePolicy<K, V> {
```

Comment:
This class is not used anywhere (yet). If you want to have "final" API without implementing the logic, you should add this to configuration, too. Or just omit the methods from `ConflictResolutionManager` for the time being.